src/standards.jl

```diff
@@ -70,33 +72,43 @@ const STANDARD_LABELS = Dict(# This EEG channel name list is a combined 10/20 an
                                          "tp9", "tp7", "cp5", "cp3", "cp1", "cpz", "cp2", "cp4", "cp6", "tp8", "tp10",
                                          "t5", "p9", "p7", "p5", "p3", "p1", "pz", "p2", "p4", "p6", "p8", "p10", "t6",
                                          "po7", "po3", "poz", "po4", "po8",
-                                         "o1", "oz", "o2",
+                                         "o1" => ["01"], "oz", "o2" => ["02"],
```

Comment:
god are we really seeing "zero-one" for `o1` in the wild??

Reply:
0 and O are right next to each other on the keyboard. From this, we can estimate the diameter of the sleep technicians fingertips.